cmd/zdb/zdb.c

```diff
@@ -2292,6 +2422,8 @@ dump_label(const char *dev)
 			nvlist_t *vdev_tree = NULL;
 
 			dump_nvlist(config, 4);
+			if (l == 3 && dump_opt['l'] >= 2)
```

Comment:
I don't think that's necessary.  I just wanted to make note of this potential issue.  I know @ofaaland was looking in to implementing this.